sdk/communication/AzureCommunication/Source/Authentication/CommunicationTokenRefreshOptions.swift

```diff
@@ -28,8 +28,8 @@ import Foundation
 
 public typealias TokenRefresherClosure = (@escaping TokenRefreshOnCompletion) -> Void
 /**
- The Communication Token Refresh Options. Used to initialize a `CommunicationUserCredential`
- - SeeAlso: ` CommunicationUserCredential.init(...)`
+ The Communication Token Refresh Options. Used to initialize a `CommunicationTokenCredential`
```

Comment:
Anything in the `AzureCommuncation` code should defer to master.

Reply:
Related to Josh's changes, the rename should go into master.